okhttp-tls/src/main/java/okhttp3/tls/HeldCertificate.kt

```diff
@@ -516,9 +517,17 @@ class HeldCertificate(
     }
 
     private fun decodePem(pem: String): X509Certificate {
-      val certificates = CertificateFactory.getInstance("X.509")
+      val certificateFactory = CertificateFactory.getInstance("X.509")
+      val certificates = certificateFactory
           .generateCertificates(Buffer().writeUtf8(pem).inputStream())
```

Comment:
@swankjesse Any reason we don't use the singular form?

https://docs.oracle.com/javase/7/docs/api/java/security/cert/CertificateFactory.html#generateCertificate(java.io.InputStream)

Reply:
Looks like it was a mistake. I like your new behavior; seems like it’d catch bugs where there are multiple certificates in a single string.